src/lecturer/components/createQuestionView/CreateQuestionView.tsx

```diff
@@ -206,9 +207,11 @@ export function CreateQuestionView() {
                 text: question,
                 options: options,
             };
+
+            store.questions = [...store.questions, obj];
```

Comment:
so if I want to load questions to the store all I need to do is to write `store questions = [array of questions]`, right?

Reply:
store.questions = questions